.gitignore

```diff
@@ -31,3 +31,4 @@ hostkey.ser
 *.h2.db
 *.factorypath
 bin
+/test.sh
```

Comment:
I don't think this has to be added to .gitignore. Why not simply avoid adding test.sh to the tracking changes? The name of the script is personal to each tester, there is nothing ensuring others will call it the same way.

Reply:
ok. Done